content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -156,7 +156,6 @@ volumeMounts:
     readOnly: true
   - mountPath: /var/log/audit.log
     name: audit-log
-    readOnly: false
```

Comment:
I don't understand this change.

Reply:
I have no idea why, but I read it readOnly: true.
Sorry for the noise this PR is not needed.